src/connection.ts

```diff
@@ -470,6 +470,9 @@ export class Connection<S extends Schema = Schema> extends EventEmitter {
    * Authorize the connection using OAuth2 flow.
    * Typically, just pass the code returned from authorization server in the first argument to complete authorization.
    * If you want to authorize with grant types other than `authorization_code`, you can also pass params object with the grant type.
+   *
+   * @returns {Promise<AuthInfo>} An object that contains the user ID, org ID and identity URL.
```

Comment:
This is `Promise<UserInfo>` right?

Reply:
in jsdoc types are inside curly brackets, we use the same syntax in sfdx-core
https://github.com/search?q=repo%3Aforcedotcom%2Fsfdx-core+%2F%40returns+%5C%7BPromise%2F&type=code

https://jsdoc.app/tags-returns.html